src/Lachain.Storage/Repositories/LocalTransactionRepository.cs

```diff
@@ -49,13 +49,10 @@ public void TryAddTransaction(TransactionReceipt receipt)
                     break;
                 }
                 
-                if (decodedRes.Length >= 5)
+                if (decodedRes[0] is UInt160 decodedAddr && decodedAddr.Equals(receipt.Transaction.To))
```

Comment:
Now you are testing only first argument. We have signatures where we should check not only first argument (transferFrom(address,address,uint256)),  this case will be missed. Just check all arguments please,  we can add some signatures in the future .